packages/wonder-stuff-core/src/keys.js

```diff
@@ -2,10 +2,10 @@
 /**
  * Return an array of the enumerable keys of an object.
  *
- * @param {$ReadOnly<{[string]: mixed}>} obj The object for which the values are
+ * @param {$ReadOnly<{[string]: V}>} obj The object for which the values are
```

Comment:
I'm curious if adding `$ReadOnly` to the type of `obj` would have fixed this issue. In any case, it seems like the doc comment should match the actual types—either the `$ReadOnly` here should be removed, or `$ReadOnly` should be added to the function signature.

Reply:
Oh, good call. Let me try that

Reply:
Yep, that works. Good spot, @benchristel. Thanks! :)